css/style.css

```diff
@@ -1,4 +1,207 @@
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  padding-top: 24px;
+  padding-bottom: 72px;
+}
+
+.header__logo-img {
+  margin-right: 258px;
+  padding-left: 43px;
+}
+
+.menubar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.menubar__link {
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+  white-space: nowrap;
+}
+
+.menubar__link:hover, 
+.menubar__link:active, 
+.menubar__link:focus {
+  cursor: pointer;
+  color: #e51937;
+  box-shadow: 0 2px 0 #e51937;
+  opacity: 0.75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+  margin-right: 63px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-right: 25px;
+}
+
+.menubar__link:nth-child(3) {
+  margin-right: 44px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-right: 46px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-right: 223px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-right: 30px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-right: 30px;
+}
+
+.button {
+  display: block;
+  background-color: #fff;
+  width: 24px;
+  height: 15px;
+  position: relative;
+  transition: all 0.5s ease 0s;
+}
+
+.button:nth-child(8) {
+  margin-right: 60px;
+}
+
+.button__line {
+  position: absolute;
+  width: inherit;
+  height: 2px;
+  background-color: #e51937;
+}
+
+.second {
+  top: 7px;
+}
+
+.third {
+  top: 14px;
+}
+
+.button:hover, 
+.button:focus, 
+.button:active {
+  transform: rotate(90deg);
+  cursor: pointer;
+  transition: all 0.5s ease 0s;
+}
+
+.content {
+  position: relative;
+  background: url("../img/jpg/Tesla\ logo\ car.jpg");
+  background-repeat: no-repeat;
+  margin-right: -8px;
+}
+
+.content__main-title, 
+.content__title {
+  text-align: center;
+  margin: 0;
+}
+
+.content__main-title {
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: -3px;
+}
+
+.content__title {
+  font-weight: 600;
+  font-size: 16px;
+  padding-bottom: 26px;
+}
+
+.content__data-car {
+  display: flex;
+  margin: 470px 0px 0 278px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content__facts {
+  display: flex;
+  margin-right: 135px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.data__speed-icon {
+  width: 54px;
+  height: 42px;
+}
+
+.content__road-icon {
+  width: 40px;
+  height: 59px;
+}
+
+.data__speed-block {
+  padding-right: 100px;
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 12px;
+}
+
+.testdrive__img {
+  width: 128px;
+  height: 30px;
+  margin-right: 40px;
+}
+
+.order-btn {
+  margin-right: 278px;
+  border: 2px solid #e51937;
+  width: 228px;
+  height: 40px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  transition: all 0.5s ease 0s; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)